knowledge/faq.rst

```diff
@@ -109,3 +109,18 @@ This error can happen, if there are no or false authentication credentials in th
     http.client.HTTPConnection.debuglevel = 1
     
 One source of error can be the ``.netrc`` file, which is `honored by the requests library <https://requests.readthedocs.io/en/latest/user/quickstart/#custom-headers>`_.
+
+
+ERROR: Obtaining different revisions in Linux and Windows
```

Comment:
is it actually an error in client? I think that's just having two different revisions, and conan 2.0 can handle that just fine.

Reply:
It is not an actual error message, but a common situation that users perceive as an error, so better have an entry here in the FAQ